index.d.ts

```diff
@@ -77,6 +84,93 @@ declare const terminalImage: {
 			preserveAspectRatio?: boolean;
 		}>
 	) => Promise<string>;
+
+	/**
+	Display GIFs in the terminal.
+
+	Optionally, you can specify the height and/or width to scale the image.
+	That can be either the percentage of the terminal window or number of rows and/or columns.
+	Please note that the image will always be scaled to fit the size of the terminal.
+	If width and height are not defined, by default the image will take the width and height of the terminal.
+	It is recommended to use the percentage option.
+	You can set width and/or height as columns and/or rows of the terminal window as well.
+	By default, aspect ratio is always maintained. If you don't want to maintain aspect ratio, set preserveAspectRatio to false.
+	Each frame of the gif is by default logged to the terminal, overwriting the previous one. To change this behaviour, set renderFrame to a different function. To change the code run when the animation playback is stopped, set renderFrame.done to a different function.
```

Comment:
```suggestion
	Each frame of the GIF is by default printed to the terminal, overwriting the previous one. To change this behavior, set `renderFrame` to a different function. To change the code run when the animation playback is stopped, set `renderFrame.done` to a different function.
```

Not clear to the user why they would want/need to override `.done` or what they should do in it.

Reply:
I also noticed this text is not in the readme. The readme and index.d.ts should be fully in sync doc-wise.